docker-images/Dockerfile-py36

```diff
@@ -6,8 +6,6 @@ RUN apt-get update
 RUN apt-get install -y git build-essential cmake libprotobuf-dev protobuf-compiler
 COPY ./ ./TenSEAL
 WORKDIR /TenSEAL
-# 3rd party libraries
-RUN git submodule init && git submodule update
```

Comment:
Why removing from here? this isn't affected by the github action checkout

Reply:
ping @philomath213

Reply:
Same for other scripts, docker-image workflow will checkout the submodules, and Dockerfile is using `COPY ./ ./TenSEAL` to copy the content to the build container. so there's is no need to to do that here.
One more thing we can't build a docker image with different submodules version, if we checkout a submodule manually to a different header, the command `git submodule update` will checkout back to specified header in our repository.